### ISSUE TITLE
URL Heart (include for payment) button disabled even after time requirements satisfied

### ISSUE THREAD

**Issue by bradleyrichter** [brave/browser-laptop#12675]
![image](https://user-images.githubusercontent.com/13509546/35027257-9c54f1b6-fb04-11e7-88ae-82d1221b0378.png)

![image](https://user-images.githubusercontent.com/13509546/35027275-ad535ca0-fb04-11e7-868b-254cc4483fda.png)

site:
https://www.theguardian.com/cities/2018/jan/16/beginning-end-trump-real-estate-empire

**Comment by NejcZdovc** [comment]
@bsclifton what is the status of this one?

**Comment by bsclifton** [comment]
@NejcZdovc when @ryanml and I met, we found a clear way to force the update to happen.  The way we triggered the update was by doing the same action which happens on window blur:
```
state = ledgerApi.pageDataChanged(state, {}, true)
```

However, there didn't seem to be a good event or interval that we could force this update to happen on.  Unless we use setTimeout or setInterval to perform a check (which could cause performance issues), it would be hard to be exact (without lots of overhead).

I think it would be great to talk about next steps on this together 😄
